examples/sdkconfig.defaults

```diff
@@ -50,3 +50,12 @@ CONFIG_ESP32_WIFI_TASK_PINNED_TO_CORE_0=y
 
 CONFIG_MBEDTLS_DEBUG=n
 CONFIG_MBEDTLS_DEBUG_LEVEL_WARN=n
+
+CONFIG_ESP_COREDUMP_ENABLE_TO_UART=y
+CONFIG_ESP_COREDUMP_DATA_FORMAT_ELF=y
+CONFIG_ESP_COREDUMP_CHECKSUM_CRC32=y
+CONFIG_ESP_COREDUMP_ENABLE=y
+CONFIG_ESP_COREDUMP_MAX_TASKS_NUM=64
+CONFIG_ESP_COREDUMP_UART_DELAY=0
+CONFIG_ESP_COREDUMP_DECODE_INFO=y
+CONFIG_ESP_COREDUMP_DECODE="disable"
```

Comment:
Looking at the docs it looks like maybe `CONFIG_ESP_COREDUMP_DECODE_INFO` is supposed to be a value to be set for `CONFIG_ESP_COREDUMP_DECODE`?

Please disregard if I've got that wrong or if what you have done here is equivalent, but as written, I can't tell if you are intending to decode or not.

Reply:
This is for esp-idf monitor to automatically decode base64 encoded coredump (we don't support that for now)